benches/nexmark/generator/strings.rs

```diff
@@ -7,14 +7,24 @@ use rand::{distributions::Alphanumeric, Rng};
 
 const MIN_STRING_LENGTH: usize = 3;
 
+/// Return a random string of up to `max_length`.
+pub(super) fn next_string<R: Rng>(rng: &mut R, max_length: usize) -> String {
+    let len = rng.gen_range(MIN_STRING_LENGTH..=max_length);
+    rng.sample_iter(&Alphanumeric)
+        .take(len)
+        .map(char::from)
+        .collect()
```

Comment:
[`DistString::sample_string()`](https://docs.rs/rand/latest/rand/distributions/trait.DistString.html#method.sample_string) is a little nicer here